### DISCUSSION TITLE
提问，抢2024BW的票的界面时还没有这个信息，今年方舟的嘉年华和崩铁的用这个的时候有这个信息吗？

### DISCUSSION THREAD

**Discussion by dxlJill** [mikumifa/biliTickerBuy#162] [提问专区]
<img width="562" alt="屏幕截图 2024-06-26 145441" src="https://github.com/mikumifa/biliTickerBuy/assets/171417496/8a1294ad-ef82-4607-bde2-9970a318a335">

**Comment by mikumifa** [comment]
现在票的信息没出来，所以没有
